js/classloader.js

```diff
@@ -71,21 +71,32 @@ function fillColumn(colNumber) {
             };
 
             var requirementLabel = document.createTextNode(requirementName.replace(/_/g, ' '));
+           
+            //div for containing dropdowns
+            var reqEntryDiv = document.createElement('div');
+            reqEntryDiv.className = 'reqEntryDiv pull-right';
 
+            //checkbox creation
+            //dropdown and checkbox styling
             var requirementCheckbox = document.createElement('input');
             requirementCheckbox.setAttribute('type', 'checkbox');
             requirementCheckbox.id = requirementName;
             requirementCheckbox.className = classes + ' pull-right requirementMarker';
-            requirementCheckbox.style.display = 'none';
+            requirementCheckbox.style.marginRight = '5px';
```

Comment:
Can we add a class and put it in the stylesheet? In-JS styles are really hard to manage/track down where styling is coming from.